common/common-iop.mk

```diff
@@ -43,6 +43,12 @@ IOP_LIBS += -lkernel -lgcc
 %.o : %.s
 	$(IOP_AS) $(IOP_ASFLAGS) $< -o $@
 
+%.o : %.lst
+	echo "#include \"irx_imports.h\"" > build-imports.c
```

Comment:
Hmm, why hardcode irx_imports?  Are there few enough imports that we'd only really have one?

In that case, why not have a single shared imports file?

-[Unknown]